lib/addons/library.kodi.audioengine/libKODI_audioengine.cpp

```diff
@@ -224,8 +225,4 @@ void CAddonAEStream::SetResampleRatio(double Ratio)
   ((CB_AudioEngineLib*)m_Callbacks)->AEStream_SetResampleRatio(((AddonCB*)m_AddonHandle)->addonData, m_StreamHandle, Ratio);
 }
 
-void CAddonAEStream::Discontinuity()
```

Comment:
Just a note, this function was not removed on last update here and related part on Kodi was already missing.